pubspec.yaml

```diff
@@ -17,7 +17,7 @@ dev_dependencies:
   build_runner: any
   build_test: any
   build_web_compilers: any
-  json_serializable: ^2.0.0
+  json_serializable: ^3.1.0
```

Comment:
2 suggestions: json_serializable has Dart SDK `>=2.2.0 <3.0.0` which means we should probably bump the minimum declared in this package to 2.2.0. Since it's a SDK minimum bump, I'd lean toward a minor release (5.1.0-dev)

Reply:
ack

Reply:
...new APIs should be a minor release anyway

Reply:
thanks! Looks good now.